spec.md

```diff
@@ -1691,6 +1691,7 @@ by the Platform:
 | _All_ | 200 | Success | No | No |
 | _All_ | 200 with malformed response | Failure | No | No |
 | [Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances) for [Provisioning](#provisioning)/[Deprovisioning](#deprovisioning) | 200 with `"state": "failed"` | Failure | Yes | No |
+| [Polling Last Operation for Service Instances update](#polling-last-operation-for-service-instances)) | 200 with `"state": "failed"` | Failure | No | No |
```

Comment:
For consistency, could the first column be changed to:
```
[Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances) for [Updating](#updating-a-service-instance)
```

Reply:
good catch - done!